cxx-sensors/src/main/resources/cppcheck.xml

```diff
@@ -6345,7 +6345,1017 @@ The expression 'strcmp(x,"def") != 0' is suspicious. It overlaps 'strcmp(x,"abc"
     <remediationFunction>LINEAR</remediationFunction>
     <remediationFunctionGapMultiplier>1min</remediationFunctionGapMultiplier>
   </rule>
-  <!-- ########### add missing rules which seems not to be part of cppcheck errorlist ########### -->
```

Comment:
It looks like rules from the cppcheck's libraries' configurations (included into standard cppcheck installation) were never registered purposeful. Some single rules were added on demand (e.g. if missing rule were reported as defect). My change

    adds all missing rules from the libraries' configuration (cppcheck 1.82)
    fixes import scripts in order to
    ** generate the SonarQube rules from cppcheck's ones
    ** find out, which rules are missing in the current version of sonar-cxx